capstone/src/main/webapp/script.js

```diff
@@ -1 +1,261 @@
+    chart.draw(stats, options);
+}
+
+function loadGenderBarChart(list){
+    let stats = new google.visualization.DataTable();
+    
+    stats.addColumn('string', 'Gender');
+    stats.addColumn('number', 'People of each gender');
+    stats.addRows([
+        ['Male', list[0]],
+        ['Female', list[1]],
+        ['Unknown', list[2]],
+    ]);
+
+    let options = {
+        title: 'Gender Bar Chart',
+        legend: 'none',
+        colors: ['#8C47CD']
+    };
+
+    // Instantiate and draw the chart.
+    const chart = new google.visualization.BarChart(document.getElementById('gender-bar-chart'));
+    chart.draw(stats, options);
+}
+
+
+function loadSentimentVResponseTimeScatterChart(listA, listB){
+    let stats = new google.visualization.DataTable();
+    
+    stats.addColumn('number', 'Sentiment Score');
+    stats.addColumn('number', 'Response Time');
+
+    const listLength = listA.length;
+    for (let i = 0; i < listLength; i++) {
+    stats.addRows([
+        [listA[i], listB[i]],
+    ]);
+    }
+
+    let options = {
+        title: 'Sentiment V. Response Time',
+        hAxis: {title: 'Response Time'},
+        vAxis: {title: 'Sentiment Score', minValue: -1, maxValue: 1},
+        legend: 'none',
+        trendlines: { 0: {} } 
+    };
+
+    // Instantiate and draw the chart.
+    const chart = new google.visualization.ScatterChart(document.getElementById('sentiment-v-response-time'));
+    chart.draw(stats, options);
+}
+
+function loadSentimentVDaysScatterChart(listA, listB){
+    let stats = new google.visualization.DataTable();
+    
+    stats.addColumn('number', 'Sentiment Score');
+    stats.addColumn('number', 'Days');
+
+    const listLength = listA.length;
+    for (let i = 0; i < listLength; i++) {
+    stats.addRows([
+        [listA[i], listB[i]],
+    ]);
+    }
+
+    let options = {
+        title: 'Sentiment V. Days',
+        hAxis: {title: 'Days'},
+        vAxis: {title: 'Sentiment Score', minValue: -1, maxValue: 1},
+        legend: 'none',
+        colors: ['#1FC142'],
+        trendlines: { 0: {} } 
+    };
+
+    // Instantiate and draw the chart.
+    const chart = new google.visualization.ScatterChart(document.getElementById('sentiment-v-days'));
+    chart.draw(stats, options);
+}
+
+function loadAgePieChart(list){
+    let stats = new google.visualization.DataTable();
+    
+    stats.addColumn('string', 'Age');
+    stats.addColumn('number', 'Percentage');
+    stats.addRows([
+        ['18-24', list[0]],
+        ['25-34', list[1]],
+        ['35-44', list[2]],
+        ['45-54', list[3]],
+        ['55-64', list[4]],
+        ['65+', list[5]],
+        ['Unknown', list[6]],
+    ]);
+
+    let options = {
+        title: 'Age Percentages',
+    };
+
+    // Instantiate and draw the chart.
+    const chart = new google.visualization.PieChart(document.getElementById('age-pie-chart'));
+    chart.draw(stats, options);
```

Comment:
I think some of the formatting is off with indentation but eslint should fix that for you